src/locales/en/translation.json

```diff
@@ -2187,6 +2189,18 @@
     "amount": "Increase margin by:",
     "liquidationPrice": "Estimated Liquidation price:"
   },
+  "addCollateral": {
+    "title": "Add Collateral",
+    "collateralBalance": "Collateral Balance",
+    "liquidationPrice": "Liquidation Price",
+    "collateralRatio": "Collateral Ratio",
+    "amount": "Increase collateral by:",
+    "newInfo": "New details:",
+    "review": "Review Transaction",
+    "errors": {
+      "balance": "Please enter an amount equal to or less than the balance of {{asset}} in your address."
```

Comment:
```suggestion
      "balance": "Please enter an amount equal to or less than your {{asset}} balance."
```

Reply:
I would probably rephrase it this way, it sounds more natural to me than writing "the balance of RBTC in your address" (but I am not a native English speaker so you can consult it with someone more qualified :D ).